core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -933,6 +948,22 @@ private SqlLibraryOperators() {
           ReturnTypes.BIGINT_NULLABLE, OperandTypes.TIMESTAMP,
           SqlFunctionCategory.TIMEDATE);
 
+  /** BigQuery's "DATETIME_ADD(timestamp, interval) function; Behaves similarly
```

Comment:
Change `BigQuery's "DATETIME_ADD(timestamp, interval) function` to `The "DATETIME_ADD(timestamp, interval) function (BigQuery)`. I don't want to assume that no other library will ever use it.

Center the javadoc in Calcite, not BigQuery. Change "; Behaves similarly to BigQuery's TIMESTAMP_ADD because in Calcite, datetime is a type alias for timestamp." to ". Like {@code TIMESTAMP_ADD}, returns a Calcite {@code TIMESTAMP} (which BigQuery calls a {@code DATETIME})."